_posts/2016-12-21-BDD-tests-for-Hadoop-with-Cucumber-part-II.md

```diff
@@ -125,7 +131,7 @@ So, it is pretty simple - we match step string to the java regular expression in
 
 As you can see we use default values for fields, which keep our scenarios in Gherkin clean and readable.  
 
-Now you can extend these test and include output content checks or other important things. 
+Now we have the basic testing structure for acceptance tests and everyone can extend test suite and include output content checks or other important things. 
```

Comment:
Would say "we have a basic tests structure" as we talk in general here
and 'the test suite' as we go into specifics
but it's been a while since I've studied grammar and prepositions, so I might be wrong